examples/features/file_system/file_system.feature

```diff
@@ -56,4 +58,6 @@ Feature: File system extension
     And I create directory deeply/nested/directory
     Then directory deeply/nested/directory should exist
     Given I remove directory deeply/nested/directory
-    Then directory deeply/nested/directory should not exist
\ No newline at end of file
+    Then directory deeply/nested/directory should not exist
+    Given I remove directory deeply
+    Then directory deeply should not exist
```

Comment:
nitpick: add a new line at EOF when it's missing

Reply:
Thanks so much, @Crow-EH for all these reviews. I take a look to them asap.